src/Hexagon.java

```diff
@@ -5,15 +5,19 @@
     private TerrainType terrainType;
     private int tileID;
     private int level;
-    private int numVillagersOnTop;
+    private int numberOfVillagersOnTop;
```

Comment:
If a hex gets nuked, the number of villagers would be 0 but the level wouldn't be. The two wouldn't always be the same, I think.